legal-api/src/legal_api/config.py

```diff
@@ -84,7 +84,7 @@ class _Config:  # pylint: disable=too-few-public-methods
     JWT_OIDC_ALGORITHMS = os.getenv("JWT_OIDC_ALGORITHMS")
     JWT_OIDC_JWKS_URI = os.getenv("JWT_OIDC_JWKS_URI")
     JWT_OIDC_ISSUER = os.getenv("JWT_OIDC_ISSUER")
-    JWT_OIDC_AUDIENCE = os.getenv("KEYCLOAK_CLIENT_ID")
+    JWT_OIDC_AUDIENCE = os.getenv("JWT_OIDC_AUDIENCE")
```

Comment:
@argush3 According to Patrick, hopefully this should do the trick and fix the authentication issue.